scripts/generate_docs

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#
+# Check that Rust documentation is generated without any warnings.
+
+readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
+# shellcheck source=scripts/common
+source "$SCRIPTS_DIR/common"
+
+if cargo doc --document-private-items --no-deps 2>&1 | grep -iq -e '^WARNING'; then
+  echo "Warnings found when generating the docs."
```

Comment:
Note that we have some complementary logic in `build_gh_pages`, namely `cargo deadlinks`; perhaps that should be moved to this script too, and this should be renamed to `check_docs` or similar? and leave `build_gh_pages` to just generate, without doing all the checking again. Also I would keep `--document-private-items` here, but not in `build_gh_pages` (as it is already in your PR). WDYT?

Reply:
I agree. `cargo deadlinks` with the `--check-http` option gives the following error. For now I am using it without the `--check-http` option. 

```
Found invalid urls in /opt/my-project/target/doc/aggregator_backend/proto/aggregator_server/struct._Inner.html:
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#188-196: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.Instrumented.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.instrument: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.WithDispatch.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.in_current_span: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#248: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#129-131: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#393: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#166-168: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#212-217: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_current_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html: 404 Not Found
Found invalid urls in /opt/my-project/target/doc/aggregator_backend/proto/struct.Sample.html:
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_current_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.Instrumented.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#188-196: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.in_current_span: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.WithDispatch.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#212-217: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#166-168: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#393: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#248: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.instrument: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#129-131: 404 Not Found
Found invalid urls in /opt/my-project/target/doc/aggregator_backend/proto/struct.SerializedSparseVector.html:
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.in_current_span: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#212-217: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.Instrumented.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#129-131: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#248: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#188-196: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#166-168: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_current_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.WithDispatch.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#393: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.instrument: 404 Not Found
Found invalid urls in /opt/my-project/target/doc/split_grpc_server/proto/hello_world_server/struct._Inner.html:
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.in_current_span: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#212-217: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_current_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#188-196: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.WithDispatch.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#129-131: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#248: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#166-168: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.Instrumented.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.instrument: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#393: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_subscriber: 404 Not Found
Found invalid urls in /opt/my-project/target/doc/aggregator_backend/struct.AggregatorBackend.html:
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#166-168: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#212-217: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#248: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#188-196: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_current_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.Instrumented.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.in_current_span: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#393: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.instrument: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.WithDispatch.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#129-131: 404 Not Found
Found invalid urls in /opt/my-project/target/doc/split_grpc_server/struct.HelloWorldHandler.html:
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#212-217: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#188-196: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.Instrumented.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#166-168: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_current_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#393: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.WithDispatch.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.in_current_span: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#129-131: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.instrument: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#248: 404 Not Found
Found invalid urls in /opt/my-project/target/doc/split_grpc_server/proto/struct.HelloResponse.html:
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_current_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.instrument: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#129-131: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#212-217: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#248: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.Instrumented.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#166-168: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.in_current_span: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#188-196: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#393: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.WithDispatch.html: 404 Not Found
Found invalid urls in /opt/my-project/target/doc/split_grpc_server/proto/struct.HelloRequest.html:
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#393: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#188-196: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_current_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#248: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.Instrumented.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#166-168: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.in_current_span: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.instrument: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#212-217: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#129-131: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.WithDispatch.html: 404 Not Found
Found invalid urls in /opt/my-project/target/doc/split_grpc_client/proto/hello_world_client/struct.HelloWorldClient.html:
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#212-217: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.WithDispatch.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.in_current_span: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#188-196: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#129-131: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#166-168: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#393: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.Instrumented.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.instrument: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_current_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#248: 404 Not Found
Found invalid urls in /opt/my-project/target/doc/split_grpc_client/proto/struct.HelloRequest.html:
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#393: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#166-168: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#188-196: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_current_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.instrument: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.in_current_span: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#129-131: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.WithDispatch.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.Instrumented.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#248: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#212-217: 404 Not Found
Found invalid urls in /opt/my-project/target/doc/split_grpc_client/proto/struct.HelloResponse.html:
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#393: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#188-196: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#166-168: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#248: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.instrument: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#129-131: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html#method.in_current_span: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.Instrumented.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.Instrument.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/trait.WithSubscriber.html#method.with_current_subscriber: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/tracing_futures/struct.WithDispatch.html: 404 Not Found
        Unexpected HTTP status fetching https://docs.rs/tracing-futures/0.2.2/src/tracing_futures/lib.rs.html#212-217: 404 Not Found
```